Orange/preprocess/_relieff.pyx

```diff
@@ -365,6 +365,9 @@ cdef tuple prepare(X, y, is_discrete, Contingencies &contingencies):
         X[:, is_continuous] -= row_min[is_continuous]
         X[:, is_continuous] /= row_ptp[is_continuous]
     y = np.array(y, dtype=np.float64)
+    isan = np.logical_not(np.isnan(y))
```

Comment:
`isan` is nondescript. Rename to `is_defined` (likened to the surrounding code), `nonnan`, or similar.